src/date-picker/calendar-toolbar.jsx

```diff
@@ -62,31 +81,17 @@ const CalendarToolbar = React.createClass({
     }
   },
 
-  _styles() {
-    return {
-      root: {
-        position: 'relative',
-        padding: 0,
-        backgroundColor: 'inherit',
-      },
-
-      title: {
-        position: 'absolute',
-        top: '17px',
-        lineHeight: '14px',
-        fontSize: '14px',
-        height: '14px',
-        width: '100%',
-        fontWeight: '500',
-        textAlign: 'center',
-      },
-    };
-  },
-
   render() {
-    let month = DateTime.getFullMonth(this.props.displayDate);
-    let year = this.props.displayDate.getFullYear();
-    let styles = this._styles();
+    const {
+      DateTimeFormat,
+      locale,
+      displayDate,
+    } = this.props
+
+    const dateTimeFormated = new DateTimeFormat(locale, {
```

Comment:
dateTimeFormatted :+1:
Regarding custom options, we could add a property or a variable in the theme for this.